project/SparkBuild.scala

```diff
@@ -420,6 +422,16 @@ object Unidoc {
   )
 }
 
+object Scalastyle {
```

Comment:
@squito @holdenk I think you can see what I'm trying to do here with the plugin config, but it doesn't work. You guys might know how to get SBT plugins configured? I'm just trying to set scalastyleConfig to `dev/scalastyle-config.xml` instead of the default of `scalastyle-config.xml`, but `sbt scalastyle` clearly shows it's not picking up this setting.